frappe/__init__.py

```diff
@@ -2068,25 +2068,36 @@ def logger(
 	)
 
 
-def log_error(message=None, title=_("Error")):
+def log_error(title=None, message=None, reference_doctype=None, reference_name=None):
```

Comment:
Why is this order switched? There are 100s of log_error calls who will check and fix them all? 🤦

Reply:
I agree with @ankush. It's used everywhere. Will make a lot of error handling fail.